compiler/circle-execution-plan/src/CircleExecutionPlan.cpp

```diff
@@ -36,6 +36,29 @@ int entry(int argc, char **argv)
   arser.add_argument("input").help("Input circle model");
   arser.add_argument("output").help("Output circle model");
   arser.add_argument("--platform").default_value("linux").help("Platform name: linux mcu cmsisnn");
+  arser.add_argument("--allocating_mode")
+    .default_value("common")
+    .help("Buffer type name (only onert-micro option):"
+          "common - a single buffer is considered for all allocations"
+          "split - there are three buffers: for input,"
+          " for output and for intermediate tensors");
+  arser.add_argument("--runtime")
+    .default_value("onert_micro")
+    .help("Target runtime name: luci-interpreter onert-micro");
```

Comment:
This code do not contain handler of `luci-interpreter` runtime.
Do you plan to add it later or this line is obsolete?

Reply:
yes, it will be added in next PR